tests/e2e-playwright/Makefile

```diff
@@ -16,11 +16,15 @@ SIMCORE_DOT_ENV = $(abspath $(CURDIR)/../../.env)
 
 define _up_simcore
 # set some parameters to allow for e2e to run
-echo LOGIN_REGISTRATION_INVITATION_REQUIRED=0 >> $(SIMCORE_DOT_ENV)
-echo LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=0 >> $(SIMCORE_DOT_ENV)
-echo DIRECTOR_DEFAULT_MAX_NANO_CPUS=1000000000 >> $(SIMCORE_DOT_ENV)
 echo DIRECTOR_DEFAULT_MAX_MEMORY=134217728 >> $(SIMCORE_DOT_ENV)
+echo DIRECTOR_DEFAULT_MAX_NANO_CPUS=1000000000 >> $(SIMCORE_DOT_ENV)
+echo LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=0 >> $(SIMCORE_DOT_ENV)
+echo LOGIN_REGISTRATION_INVITATION_REQUIRED=0 >> $(SIMCORE_DOT_ENV)
 echo SIDECAR_FORCE_CPU_NODE=1 >> $(SIMCORE_DOT_ENV)
+# DISABLED: test setup has no otl service
+echo TRACING_OPENTELEMETRY_COLLECTOR_ENDPOINT=null >> $(SIMCORE_DOT_ENV)
+echo TRACING_OPENTELEMETRY_COLLECTOR_PORT=null >> $(SIMCORE_DOT_ENV)
```

Comment:
This is not how we disable this feature. Did you test this? We usually Set `CATALOG_TRACING=null` from `CATALOG_TRACING={}`. Did something change with Pydantic v2 and does it now work as you describe it?

Reply:
I have to disable them everywhere in the entire stack. This is how you can do it since you prevent the auto-default factory to build the settings and falls back in `None`